code/modules/spells/spell_types/pointed/coinflip.dm

```diff
@@ -0,0 +1,63 @@
+/datum/action/cooldown/spell/pointed/coinflip
+
+	name = "Fear And Hunger"
+	desc = "This spell causes the target to flip an imaginary coin, and if they call it wrong they lose a random limb"
+	button_icon_state = "autotomy"
+
+	/// Message showing to the spell owner upon activating pointed spell.
+	active_msg = "You prepare a spectral coin."
+	/// Message showing to the spell owner upon deactivating pointed spell.
+	deactive_msg = "You put the coin back in your pocket..."
+	/// The casting range of our spell
+	cast_range = 5
+	/// Variable dictating if the spell will use turf based aim assist
+	aim_assist = TRUE
+
+	sound = 'sound/items/coinflip.ogg'
+	school = SCHOOL_CONJURATION
+	cooldown_time = 120 SECONDS
+
+	invocation = "F'R ND H'NGR"
+	invocation_type = INVOCATION_WHISPER
+	spell_requirements = SPELL_REQUIRES_NO_ANTIMAGIC
+
+/datum/action/cooldown/spell/pointed/coinflip/is_valid_target(atom/cast_on)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!ishuman(cast_on))
+		return FALSE
+
+	return
+
+/datum/action/cooldown/spell/pointed/coinflip/cast(mob/living/carbon/human/cast_on)
+	. = ..()
+	if(cast_on.can_block_magic(antimagic_flags))
+		to_chat(cast_on, span_notice("You hear a coin flipping in the distance, but nothing more."))
+		to_chat(owner, span_warning("The spell had no effect!"))
+		return FALSE
+
+	to_chat(cast_on, span_warning("Heads, or tails!"))
+	cast_on.SetImmobilized(10 HOURS)
+	select_coin(cast_on)
+	
+	return TRUE
+
+
+/datum/action/cooldown/spell/pointed/coinflip/proc/select_coin(mob/living/user, title = "Heads, or Tails?")
+	var/answerChoice = tgui_input_list(user, "Heads or tails", "Call it", list("heads","tails"))
+	var/list/sideslist = list("heads","tails")
```

Comment:
Furthermore, I think this would be better if it blinded the guy and gave them a radial menu, so it would sorta be like this
![example](https://github.com/yogstation13/Yogstation/assets/53777086/e4c3bb68-6124-4a27-b73a-f5cc2a6e006a)

Reply:
Or if you want to go even further you can put a nodrop coin in their hand that they have to flip.

Reply:
this is actually a pretty solid idea

Reply:
but can't you still click out of radial menus without having to choose an option? Or is there a way to force keep it open, or I guess if they click out cause the flip to happen anyways

Reply:
same as if they refuse to pick an option in the UI, you can make it automatically select an option after x amount of time instead.